tests/test_swagger.py

```diff
@@ -0,0 +1,319 @@
+import operator
+
+from marshmallow import fields, Schema
+import pytest
+
+from flask_resty import (
+    Api, Filtering, GenericModelView, PagePagination, Sorting,
+    IdCursorPagination,
+)
+from flask_resty.spec import ModelViewDeclaration
+
+# -----------------------------------------------------------------------------
+
+try:
```

Comment:
I have this for the JWT tests because I run the tests both with and without JWT handling installed.